phoenicis-javafx/src/main/java/org/phoenicis/javafx/components/common/skin/IconsListElementSkin.java

```diff
@@ -0,0 +1,113 @@
+package org.phoenicis.javafx.components.common.skin;
+
+import javafx.beans.Observable;
+import javafx.beans.binding.Bindings;
+import javafx.css.PseudoClass;
+import javafx.scene.control.Label;
+import javafx.scene.control.Tooltip;
+import javafx.scene.effect.ColorAdjust;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.scene.shape.Rectangle;
+import org.phoenicis.javafx.components.common.control.IconsListElement;
+
+/**
+ * The skin for the {@link IconsListElement} component
+ *
+ * @param <E> The concrete type of the element shown in this list element
+ */
+public class IconsListElementSkin<E> extends SkinBase<IconsListElement<E>, IconsListElementSkin<E>> {
+    private static final PseudoClass SELECTED_PSEUDO_CLASS = PseudoClass.getPseudoClass("selected");
+
+    /**
+     * Constructor for all SkinBase instances
+     *
+     * @param control The control belonging to the skin
+     */
+    public IconsListElementSkin(IconsListElement<E> control) {
+        super(control);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void initialise() {
+        final Label title = createTitle();
+        final Region miniature = createMiniature();
+
+        final VBox container = new VBox(miniature, title);
+        container.getStyleClass().add("iconListElement");
+
+        container.widthProperty().addListener((observable, oldValue, newValue) -> container
+                .setClip(new Rectangle(container.getWidth(), container.getHeight())));
+
+        container.heightProperty().addListener((observable, oldValue, newValue) -> container
+                .setClip(new Rectangle(container.getWidth(), container.getHeight())));
+
+        // update the selected pseudo class according to the selected state of the component
+        getControl().selectedProperty().addListener((Observable invalidation) -> container
+                .pseudoClassStateChanged(SELECTED_PSEUDO_CLASS, getControl().isSelected()));
+        // adopt the selected state during initialisation
+        container.pseudoClassStateChanged(SELECTED_PSEUDO_CLASS, getControl().isSelected());
+
+        getChildren().addAll(container);
+    }
+
+    /**
+     * Creates a label for the title of the icon list element
+     *
+     * @return A label with the title of the list element
+     */
+    private Label createTitle() {
```

Comment:
We should have a common nomenclature (either label or title).

Reply:
You choose :)

Reply:
Then keep the previous naming (otherwise you have to rename the CSS class as well).